src/radical/pilot/task_description.py

```diff
@@ -584,7 +584,7 @@ class TaskDescription(ru.TypedDict):
         RAPTOR_ID       : str         ,
         RAPTOR_FILE     : str         ,
         RAPTOR_CLASS    : str         ,
-        METADATA        : None        ,
+        METADATA        : {str: None} ,
```

Comment:
should we also provide an empty dictionary as a default value instead of `None`?

Reply:
Yeah, probably because we do the same on other attributes.  It's probably stupid to carry that many empty entities around, but oh well... :-)  Fixed!